packages/kbn-build/src/production/index.test.js

```diff
@@ -0,0 +1,37 @@
+import tempy from 'tempy';
+import copy from 'recursive-copy';
+import { resolve } from 'path';
+import globby from 'globby';
+
+import { createProductionBuild } from './';
+import { getProjects } from '../utils/projects';
+
+test(
+  'foo',
```

Comment:
This is just a temp test to have faster feedback loop. Will clean up and make it into a proper test when we've agreed on approach.

Reply:
I think it's safe to say we're good on approach

Reply:
Awesome. I'll start working on a longer description/readme for this + improve the test.